source/documentation/virtual_machine_management_guide/topics/Affinity_groups.adoc

```diff
@@ -21,7 +21,7 @@ For this module to work, the *weight module* section of the scheduling policies
 .The Enforcing Option and Filter Module
 Both rules have an *Enforcing* option which applies the _filter module_ in the cluster's scheduling policy. The filter module overrides the weight module. With the filter module enabled, the scheduler _requires_ that a rule be fulfilled. If a rule cannot be fulfilled, the filter module prevents the virtual machines in the affinity group from running.
 
-For example, with a positive *Host Affinity Rule* and *Enforcing* enabled (the filter module enabled), the scheduler _requires_ the affinity group's virtual machines to run on hosts that are part of the affinity group. However, if those hosts are down, the scheduler does not run the virtual machines at all.
+For example, with a *Positive Host Affinity Rule* and *Enforcing* enabled (the filter module enabled), the scheduler _requires_ the affinity group's virtual machines to run on hosts that are part of the affinity group. However, if those hosts are down, the scheduler does not run the virtual machines at all.
 
 For this module to work, the *filter module* section of the scheduling policies must contain the `VmAffinityGroups` and `VmToHostsAffinityGroups` keywords.
```

Comment:
if the UI says "filter module" rather than "Filter module", consider filing a UI bug.